### ISSUE TITLE
[Feature request] Allow percentages/px in bgImg position

### ISSUE THREAD

**Issue by paulrudy** [delucis/astro-og-canvas#75]
Thanks for the awesome package! At some point, would you consider adding the ability to specify `%` (and/or `px`) values for the `bgImg` position?

This would permit being able to specify a per-image offset—for example, in frontmatter.

**Comment by delucis** [comment]
Yeah, potentially. Currently this is designed with the assumption that `bgImage` is a most likely a tailored graphic designed to be rendered across the full OG image, but I can see wanting to allow some way to position it too.
